source/configure/environment-configuration-settings.rst

```diff
@@ -3643,6 +3644,95 @@ Allow untrusted internal connections
 |   but not ``10.0.16.20``.                                                                                                                     |
 +-----------------------------------------------+-----------------------------------------------------------------------------------------------+
 
+Mobile security
+---------------
+
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
+
+From Mattermost v10.7 and mobile app v2.27, you can configure biometric authentication, prevent Mattermost use on jailbroken or rooted devices, and can block screen captures without relying on an EMM Provider. Configure these options by going to **System Console > Environment > Mobile Security**, or by editing the ``config.json`` file as described in the following tables. Changes to configuration settings in this section require a server restart before taking effect.
+
+.. config:setting:: enable-biometric-authentication
+  :displayname: Enable Biometric Authentication
+  :systemconsole: Environment > Mobile Security
+  :configjson: .NativeAppSettings.MobileEnableBiometrics
+  :environment: MM_NATIVEAPPSETTINGS_MOBILEENABLEBIOMETRICS
+  :description: Enforces biometric authentication (with PIN/passcode fallback) before accessing the app. Users will be prompted based on session activity and server switching rules.
+
+    - **true**: Biometric authentication is enabled.
+    - **false**: **(Default)** Biometric authentication is disabled.
+
+Enable biometric authentication
```

Comment:
Is Telemetry being collected for these new configs?

Reply:
Great question! The short answer is no. But that opens another topic: do we want to?

@marianunez Do you know if we want to collect telemetry on these options? The telemetry I am thinking of is just the configuration tracking we have for other configs (see https://github.com/mattermost/mattermost/blob/c049748b88636196d76bc5a7ca8b0bfc65cad368/server/platform/services/telemetry/telemetry.go#L473).

Reply:
The server configuration is already tracked overall in telemetry but this new settings would need to be added here; https://github.com/mattermost/mattermost/blob/c049748b88636196d76bc5a7ca8b0bfc65cad368/server/platform/services/telemetry/telemetry.go#L473

That being said, we are not relying much on Rudder data anymore so 0/5 if we add it as I don't know if we will be analyzing it as we don't have dashboards to do so and are looking to move on from this model.